.github/workflows/test-ci-prod.yaml

```diff
@@ -58,6 +58,15 @@ jobs:
         env:
           GOOGLE_APPLICATION_CREDENTIALS: ./packages/backend/serviceAccountKey.json
 
+      # run coveralls
+      - name: Coveralls
+        uses: coverallsapp/github-action@master
+        with:
+            github-token: ${{ secrets.GITHUB_TOKEN }}
+            flag-name: run-${{ matrix.type }}
```

Comment:
Remove this input. This workflow doesn't have `matrix.type` value

Reply:
Ohh correct, I didn't see the `lcov.info` location under `coverage` without the need of `matrix.type`. Thanks for the heads up!